### PR TITLE
Production sync of update-filesize changes

### PR DESCRIPTION
**Pull request by mjangda**
- Fix index bug that was causing batches to be skipped
- `get_attachments` only returns attachments without filesize for faster iteration through jobs
- `update_attachments` now returns counts of the various results
- use a HEAD request and header sniffing to get the content size (few requests and faster)
- Speed up the jobs by increasing batch size and frequency
- Add more detail to output

### CODE REVIEW

File: files/class-vip-filesystem.php [hunk 1]
```diff
@@ -494,16 +495,20 @@ public function update_attachment_meta() {
 			return;
 		}
 
-		$updater = new Meta_Updater( 1000 );
+		$batch_size = 3000;
```
Comment (hanifn) [inline files/class-vip-filesystem.php]: I notice that this change doesn't include the batch size modifications for specific sites

File: files/class-vip-filesystem.php [hunk 2]
```diff
@@ -433,8 +433,9 @@ public function filter_cron_schedules( $schedule ) {
 			return $schedule;
 		}
 
+		// Not actually five minutes; we want it to run faster though to get through everything.
 		$schedule['vip_five_minutes'] = [
-			'interval' => 300,
+			'interval' => 180,
 			'display' => __( 'Once every 5 minutes' ),
```
[the remark this answers is not in the document]
Comment (mjangda) [reply files/class-vip-filesystem.php]: I don't think we can easily change the slug but can definitely change the display.

File: files/class-vip-filesystem.php (hunk 1 above)
Comment (mjangda) [reply files/class-vip-filesystem.php]: Yeah, was trying to avoid referencing clients directly since this is a public repo.

Comment (hanifn) [reply files/class-vip-filesystem.php]: AH that makes sense

**Comment by mjangda** [comment]
r136219-deploy

**Pull request merged**